code/modules/assembly/signaler.dm

```diff
@@ -250,3 +251,14 @@
 	return
 /obj/item/assembly/signaler/cyborg/screwdriver_act(mob/living/user, obj/item/I)
 	return
+
+/obj/item/assembly/signaler/syndicate
+	desc = "Used to remotely activate devices. Allows for syncing when using a secure signaler on another. This one feels unusually light."
+	quiet = TRUE
+	item_state = "signaler_syndicate"
+	icon_state = "syndicate_off"
+
+/obj/item/assembly/signaler/syndicate/signal()
+	..()
+	flick("syndicate_signaler_animated", src) 
+	
```

Comment:
```suggestion

/obj/item/assembly/signaler/syndicate/receive_signal(datum/signal/signal)
	. = FALSE
	if(!signal)
		return
	if(signal.data["code"] != code)
		return
	if(!(src.wires & WIRE_RADIO_RECEIVE))
		return
	if(suicider)
		manual_suicide(suicider)
		return
	pulse(TRUE)
```

Reply:
if you do it like this you won't need an if check as it will instead execute this proc and not the proc from the parent so there will not be the need for any changes for that it also removes the need for the quite var define.

Reply:
Instead the proc from the parent will be overriden and only this code will actually get executed on the proc call.